packages/communication-react/review/communication-react.api.md

```diff
@@ -1047,13 +1050,10 @@ export const DEFAULT_COMPOSITE_ICONS: {
     NoticePageJoinCallFailedDueToNoNetwork: JSX.Element;
     NoticePageLeftCall: JSX.Element;
     NoticePageRemovedFromCall: JSX.Element;
+    AttachFile: JSX.Element;
```

Comment:
This name is too generic. We might have other places where we attach a file in the future. Could we name this `MessageThreadAttachFile` or something with a bit more context?

Reply:
> Looks like we lost the left-alignment of `SendBox` and the typing indicator text when file attach button is missing: ![image](https://user-images.githubusercontent.com/82062616/151428367-06a78f24-919d-4a76-af58-f24b9f269116.png)
> 
> Also, why do the snapshots for `ErrorBar` tests not have the file attach button?

Filed a bug for the icon not rendering on some pages. It's a weird bug that @JamesBurnside  also noticed.

I am unable to find this Ui snapshot that you referenced here...
What's the name of the file?

Reply:
Found the image, fixed itr